editor/plugins/lightmap_gi_editor_plugin.cpp

```diff
@@ -103,6 +103,9 @@ void LightmapGIEditorPlugin::_bake_select_file(const String &p_file) {
 			case LightmapGI::BAKE_ERROR_FOREIGN_DATA: {
 				EditorNode::get_singleton()->show_warning(TTR("Lightmap data is not local to the scene."));
 			} break;
+			case LightmapGI::BAKE_ERROR_LIGHTMAP_TOO_SMALL: {
+				EditorNode::get_singleton()->show_warning(TTR("Failed creating lightmap images. Make sure all meshes selected to bake have [code]lightmap_size_hint[/code] value set high enough, and [code]texel_scale[/code] value of LightmapGI is not too low."));
```

Comment:
`[code]` tags are not supported in warning dialogs. Use backticks instead as a fallback.

Reply:
Done.